datacenter/dtr/2.2/guides/admin/upgrade/index.md

```diff
@@ -43,6 +74,16 @@ By default the upgrade command runs in interactive mode and prompts you for
 any necessary information. You can also check the
 [reference documentation](../../../reference/cli/index.md) for other existing flags.
 
+The upgrade command will start replacing every container in your DTR cluster,
+one replica at a time. It will also perform certain data migrations. If anything
+fails or the upgrade is interrupted for any reason, you can re-run the upgrade
+command and it will resume from where it left off.
+
+## Patch upgrade
+
+A patch upgrade changes only the DTR containers and it should be a relatively
+safe operation. The command is the same as for a minor upgrade.
```

Comment:
I see a lot of sentences like that in docs related to install, upgrade, and security. Generally I agree with @alexmavr and avoid sentences like that altogether.
When we say "this is safer", "there are risks" or things like that:

* Expert users will know it (probably better than us), so they won't learn anything
* Novice users will be scared because now we're telling them that something can go wrong, but won't know what to do, since we're not teaching them the subtle differences and what to do in each case.

So in most cases both users are better off if we don't even include that sentence.